release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,23 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 09 September 2024 - Enhancement - Catalog - Catalog item versions with no artifact available in the cloud will contain a message [ID_40616]
+
+When a Catalog item version is not available to deploy because its installation file is not registered, a message will be shown to the user as not being available and a request to contact support.cloud.ecosystem@skyline.be
+
+#### 09 September 2024 - New Feature - Catalog API - Catalog Registration supports images in description readme file [ID_40219]
+
+Catalog item registration now supports images to be included in the provided reamde.md file that will be used as the description of a catalog item. Any used images need to be included in a "Images" directory.
+supported image formats are jpg, jpeg, png, gif, bmp, tif, tiff and webp.
+
+#### 09 September 2024 - Enhancement - Catalog API - Catalog Registration API has been improved [ID_40670]
+
+The previously available catalog API call to register a new catalog item (version) using a manifest yaml file has been split up into having a PUT call available to register an item (including its description) and a POST call to register a version.
+
+#### 09 September 2024 - New Feature - Catalog - Unauthenticated access [ID_40686] [ID_40573] [ID_40572] [ID_40571] [ID_40570]
+
+As an unauthenticated user, it is now possible to search public available catalog items, view their description and their available versions.
```

Comment:
As an unauthenticated user, it is now possible to search publicly available catalog items, view their description and their available versions.

Reply:
ok, adapted